frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -46,15 +43,18 @@
 import org.apache.commons.cli.Options;
 import org.apache.commons.io.IOUtils;
 import org.pytorch.serve.archive.model.Manifest;
+import org.pytorch.serve.device.SystemInfo;
 import org.pytorch.serve.metrics.MetricBuilder;
 import org.pytorch.serve.servingsdk.snapshot.SnapshotSerializer;
 import org.pytorch.serve.snapshot.SnapshotSerializerFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class ConfigManager {
-    // Variables that can be configured through config.properties and Environment Variables
-    // NOTE: Variables which can be configured through environment variables **SHOULD** have a
+    // Variables that can be configured through config.properties and Environment
+    // Variables
```

Comment:
Looks like we have added unnecessary formatting changes, we should align with upstream

Reply:
Reverted the unnecessary formatting change